apps/mobile/src/libs/packages/toast-message/toast-message.package.ts

```diff
@@ -0,0 +1,44 @@
+import Toast from "react-native-toast-message";
+import { type ValueOf } from "shared";
+
+import { ToastMessageTitle, ToastMessageType } from "~/libs/enums/enums";
+
+import { DEFAULT_ERROR_MESSAGE } from "./libs/constants/constants";
+import { typeToTitle } from "./libs/maps/maps";
+
+type ErrorProperties = {
+	message?: string | undefined;
+	title?: ValueOf<typeof ToastMessageTitle>;
+};
+
+type Properties = {
```

Comment:
And I think we can leave only this type. We don't want to show toast messages without message